packages/palette/src/elements/Modal/ModalBase.tsx

```diff
@@ -1,9 +1,9 @@
-import React, { useEffect, useRef, useState } from "react"
+import React, { useEffect, useRef } from "react"
 import { createPortal } from "react-dom"
 import { RemoveScroll } from "react-remove-scroll"
 import styled from "styled-components"
 import { zIndex as systemZIndex, ZIndexProps } from "styled-system"
-import { useCursor } from "use-cursor"
+import { useFocusLock } from "../../utils/useFocusLock"
```

Comment:
I wonder if we could rename `utils` to `hooks` now, as that's mostly whats in there

Reply:
I think that makes sense. There's one function `splitProps` which is a util; so we could just have a new hooks dir and move those. Will be a separate PR.